install/helm/open-match/templates/_helpers.tpl

```diff
@@ -104,8 +104,8 @@ livenessProbe:
     scheme: {{ if (.isHTTPS) }}HTTPS{{ else }}HTTP{{ end }}
     path: /healthz
     port: {{ .port }}
-  initialDelaySeconds: 5
-  periodSeconds: 5
+  initialDelaySeconds: 10
```

Comment:
What behavior does this change? Would the user ever need to tweak these? in what scenario?

Reply:
It changes the health check frequency to 10 seconds per call. This can somewhat alleviate the spam from `/healthz` when we turn on the rpc logging. Also 10 is the default value for k8s probes check.